src/content/en/ai-capabilities/line-art-coloring.md

```
@@ -0,0 +1,62 @@
+
+It is the task of adding color to line art.
+
+For those who can already draw line art themselves, "how to make AI color my line art" is often more important than text2image from scratch.
+
+---
+
+## Line Art Coloring with ControlNet
+
+By using ControlNet Canny (edge), you can add color while generally retaining the input line art.
+
+![](https://gyazo.com/7e1ed10e17831b224bc547a8d6b3deea){gyazo=image}
```

markdownlint-cli2 (0.18.1)

Images should have alternate text (alt text)

(MD045, no-alt-text)